models/Task.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require("mongoose")
+
+const TaskSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: [true, "must provide a name"],
+    trim: true,
+    maxLength: [20, "name cannot be more than 20 characters"],
```

Comment:
20 characters?? Think our most important customer: John-Michael Williams Jameson III!

Research the process of migrating an existing MongoDB database to change the schema on existing data & bonus points if you do the migration on your MongoDB instance!

Reply:
🤣